integration_tests/cairozero_test.go

```diff
@@ -76,18 +80,27 @@ func TestCairoZeroFiles(t *testing.T) {
 			continue
 		}
 
+		start := time.Now()
+
 		pyTraceFile, pyMemoryFile, err := runPythonVm(dirEntry.Name(), compiledOutput)
 		if err != nil {
 			t.Error(err)
 			continue
 		}
 
+		elapsed_py := time.Since(start)
```

Comment:
can you benchmark before and after `	res, err := cmd.CombinedOutput()` ?  which is the real run

as runVM and runPythonVM are not strictly the same

Reply:
Hmmm... Can you point out the differences. From what I can see, Rest are constant time operations only and should not add any significant overhead. Even if not so, I think both the functions are sufficiently similar to have the same overheads.

Reply:
why not benchmarking only the hint? you benchmark additionnal code without any reason right now

Reply:
Hmm... Okay will make the changes. My point was that the current impl would not make much difference since the operations other than the main operation take place in constant negligible time.

Reply:
done.